libfwupd/meson.build

```diff
@@ -192,6 +192,8 @@ if get_option('tests')
   env = environment()
   env.set('G_TEST_SRCDIR', meson.current_source_dir())
   env.set('G_TEST_BUILDDIR', meson.current_build_dir())
+  # ensure `diff` does not fail because of locale specific decimal separator
+  env.set('LANG', 'C')
```

Comment:
Don't we want an actual locale `en_US.UTF-8`, no?

Reply:
@superm1 do you have any wisdom here?

Reply:
I'd be fine with `C.UTF-8` -- could you sprinkle it around for the other test binaries too please.

Reply:
Yeah, we need to be consistent that this is done for the env for all our test binaries.